Directory.Build.targets

```diff
@@ -18,12 +18,15 @@
   <Import Project="eng\Baseline.Designer.props" />
 
   <PropertyGroup Condition=" '$(IsPackable)' != 'false' AND '$(IsServicingBuild)' == 'true' ">
-    <IsPackable>$(PackagesInPatch.Contains(' $(PackageId);'))</IsPackable>
+    <IsPackageInThisPatch Condition="'$(IsPackageInThisPatch)' == ''">$(PackagesInPatch.Contains(' $(PackageId);'))</IsPackageInThisPatch>
+    <!-- Suppress creation of .nupkg for servicing builds. -->
+    <IsPackable Condition=" '$(IsPackageInThisPatch)' != 'true' ">false</IsPackable>
```

Comment:
These two lines look like a nice optimization. Could they increase the risk of this change?

Reply:
Unlikely. I did this to improve readability, and also to align this section of code with whats in aspnet/AspNetCore. But if you spot a problem with it, let me know.